src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -24,8 +24,11 @@ const {
 const ButtonGroup = createClass({
 	displayName: 'ButtonGroup',
 
-	childProps: {
-		Button: { ...Button.propTypes }
+	components: {
+		Button: createClass({
```

Comment:
you're right we should reference it as is right here instead of creating a new type, but the goal of this change was a 1:1 change in how child components are defined and filtered, we should have another PR that _cleans_ up any redundancies that we identify with child components